boards/arm/efr32_radio/board.cmake

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(openocd)
+board_runner_args(jlink "--device=EFR32FG1PxxxF256")
```

Comment:
I believe this would have to be change depending on the daughter board and the used efr32 soc on it.

Reply:
Good catch. I remembered I need to update it but then I didn't and tested with OpenOCD only. Fixed and tested with both runners.